packages/ui/src/containers/AppNotification/index.module.scss

```diff
@@ -21,5 +21,6 @@
     top: _.unit(-6);
     transform: translate(-50%, -100%);
     max-width: 520px;
+    white-space: nowrap;
```

Comment:
why did it wrap? should we let it wrap when it reaches the max width?

Reply:
Makes sense. Let me change it to an alternative way

Reply:
Now the text still wraps when it is too long to be contained in 520px width
<img width="669" alt="image" src="https://user-images.githubusercontent.com/12833674/176950314-ef0c1edd-0547-466f-9b7d-1c9ca47deeac.png">
But it does not wrap when it is not needed

Reply:
cool

Reply:
just double check: is the fixed width 520px expected?

Reply:
Maybe yes, as the sign in form has a fixed width, it makes sense to also have the upper notification width fixed.

Reply:
You are right. Modified to `width: fit-content` to avoid reaching and being fixed at max-width